codegen/aws/core/build.gradle.kts

```diff
@@ -12,4 +12,5 @@ extra["moduleName"] = "software.amazon.smithy.python.aws.codegen"
 dependencies {
     implementation(project(":core"))
     implementation(libs.smithy.aws.traits)
+    implementation("org.jsoup:jsoup:1.19.1")
```

Comment:
The version should go in [here](https://github.com/smithy-lang/smithy-python/blob/develop/codegen/gradle/libs.versions.toml). Docs on how that works are [here](https://docs.gradle.org/current/userguide/version_catalogs.html).